samples/js/picasa/slurp/src/main.js

```diff
@@ -158,15 +161,15 @@ async function main(): Promise<void> {
   // necessary. Consider doing something about that.
 
   const j = (v: any) => jsonToNoms(v);
-  return db.commit(out, newStruct('', {
+  return db.commit(out, newStructWithType(makeStructType('', {value: valueType}), [newStruct('', {
```

Comment:
Maybe explain why you need to do this, and in this particular way? Also assigning the `newStruct(...)` portion of this to a variable would make it easier for me to read, but up to you.